eng/configure-toolset.sh

```diff
@@ -2,4 +2,7 @@
 # We could potentially try to find an existing installation that has all the required runtimes,
 # but it's unlikely one will be available.
 
-useInstalledDotNetCli="false"
\ No newline at end of file
+useInstalledDotNetCli="false"
+
+# Working around issue https://github.com/dotnet/arcade/issues/7327
+DisableNativeToolsetInstalls=true
```

Comment:
Despite the linked issue being closed, this fix is STILL ACTIVE. Meaning, it actually does work around the reported issue.